src/index.js

```diff
@@ -6,83 +6,202 @@ const { ApolloServer, gql } = require('apollo-server');
 const books = [
   {
     id: 1,
-    title: 'Harry Potter and the Chamber of Secrets',
+    name: 'Harry Potter and the Chamber of Secrets',
     author: 1,
+    tags: ['SciFi', 'Adventure'],
   },
   {
     id: 2,
-    title: 'Jurassic Park',
+    name: 'Jurassic Park',
     author: 2,
+    tags: ['SciFi'],
   },
   {
     id: 3,
-    title: 'The Cave',
+    name: 'The Cave',
     author: 3,
+    tags: ['Adventure'],
   },
 ];
 
 const users = [
   {
     id: 1,
-    name: 'J.K. Rowling',
+    firstName: 'J.K.',
+    lastName: 'Rowling',
     address: 'U.K.',
-    favoriteBook: 1,
+    dob: '21/03/1980',
+    interests: ['SciFi', 'Adventure'],
+    isActive: true,
+    favoriteBooks: [1],
   },
   {
     id: 2,
-    name: 'Michael Crichton',
+    firstName: 'Michael',
+    lastName: 'Crichton',
     address: 'USA',
-    favoriteBook: 2,
+    dob: '04/03/1970',
+    interests: ['SciFi'],
+    isActive: true,
+    favoriteBooks: [1, 2],
   },
   {
     id: 3,
-    name: 'Batman',
+    firstName: 'Batman',
+    lastName: '',
     address: 'Gotham',
-    favoriteBook: 1,
+    dob: '21/03/1988',
+    interests: ['Adventure'],
+    isActive: true,
+    favoriteBooks: [1, 3],
   },
   {
     id: 4,
-    name: 'Spiderman',
+    firstName: 'Spiderman',
+    lastName: 'Amazing',
     address: 'New York',
-    favoriteBook: 2,
+    dob: '23/07/1992',
+    interests: ['SciFi', 'Adventure'],
+    isActive: false,
+    favoriteBooks: [1, 2, 3],
+  },
+];
+
+const publications = [
```

Comment:
This is not used and can be deleted - see failing travis build. https://travis-ci.com/gr8guyrabi/library-backend/builds/91749406#L476